docs/config.md

```diff
@@ -81,9 +81,23 @@ labels:
 
 Please refer to the [gRPC guide](./grpc.md).
 
-## (WIP) Justification
+## Justification
```

Comment:
It comes to be that we should group justification configs. E.g.

```yaml
justification:
  enabled: true
  jvs_jwks_endpoint: https://example.com. # this should be a required field if justification is enabled
```

Reply:
We also need to point out this is only meaningful for gRPC services (not http)

Reply:
Actually since it's gRPC only, we should move this to https://github.com/abcxyz/lumberjack/blob/main/docs/grpc.md

Reply:
> It comes to me that we should group justification configs. E.g.
> 
> ```yaml
> justification:
>   enabled: true
>   jvs_jwks_endpoint: https://example.com. # this should be a required field if justification is enabled
> ```

https://github.com/abcxyz/lumberjack/blob/d4bc50031aec168f9fdcf23d4d7217fc8ed0c4fb/clients/go/apis/v1alpha1/config_test.go#L58
This is not what the code does right now, do we want to change it?

Reply:
I would prefer changing it now. We can punt this PR until we make that change. In fact, I think we should fix the justification processor as we discussed before writing the doc.

Reply:
done. Still change it here instead of https://github.com/abcxyz/lumberjack/blob/main/docs/grpc.md since it's no longer grpc only after merge of #283